### ISSUE TITLE
[FEAT.] : Add the ability to filter out already installed maps

### ISSUE THREAD

**Issue by Quietess** [Zagrios/bs-manager#348]
Maps downloader is great, but having to scroll through the tons of maps I've already installed makes it hard to find new stuff I haven't already tried in niche tags.

**Comment by Snailedlt** [comment]
I'm working on this rn, I'm thinking the implementation can look like this: 
![image](https://github.com/Zagrios/bs-manager/assets/43886029/e3734dc3-2430-437b-be42-2603289c4afd)

What do you think?

**Comment by Zagrios** [comment]
Yes it can work 
But I have some questions with this UI, when its unchecked you include both owned and not owned maps ? And when its checked only the owned ones ?
I think it would be more usefull if the can filter only not owned maps, but idk if it can work with this UI 🤔

**Comment by Snailedlt** [comment]
@Zagrios The thought of this design was: 
If `owned` is checked, both owned and unowned is shown. If it's unchecked only unowned are shown.

We could however change it to the same filter state as Tags, so that it has three states:
1. only show owned games
2. only show unowned games
3. show both owned and unowned games

Keep in mind that we'll have to have different default values here for the local maps and download maps views, so that we don't end up showing only unowned maps in our local map view, or only owned maps in the download maps view.

**Comment by Zagrios** [comment]
I think I prefer the solution with the Tag, maybe to put under the title `Other` under the tags list 🤔 
Uncheked by default and hidden when this filter panel is used in the local maps list (througt a props) ?

**Comment by Snailedlt** [comment]
Yeah, sounds good to me 👍

**Comment by Snailedlt** [comment]
@Zagrios Something like this right?
![image](https://github.com/Zagrios/bs-manager/assets/43886029/e66aa6cf-55e2-44aa-86a5-0c20624ab02d)

**Comment by Zagrios** [comment]
@Snailedlt 
Yup, maybe try to add `OTHER` title above the other tags? See if it works well, otherwise don't put it
